src/types/spot.ts

```diff
@@ -558,7 +558,7 @@ export interface SymbolExchangeInfo {
   allowedSelfTradePreventionModes: SelfTradePreventionMode[];
 }
 
-export interface ExchangeInfo {
+export interface ExchangeInfoSpot {
```

Comment:
This is a breaking change, pls undo. It's been `ExchangeInfo` in master for quite some time:
https://github.com/tiagosiebler/binance/blob/master/src/types/spot.ts#L561